erpnext/manufacturing/doctype/job_card/job_card.js

```diff
@@ -70,9 +70,10 @@ frappe.ui.form.on('Job Card', {
 						fieldname: 'employee'}, d => {
 						if (d.employee) {
 							frm.set_value("employee", d.employee);
+						} else {
+							frm.events.start_job(frm);
```

Comment:
```suggestion
						}
						frm.events.start_job(frm);
```

Row should be added even after employee is set

Reply:
Earlier a row was added when employee is set, which created double entries for stock quantities on completion. 
The above fix is as per the erpnext upstream fix for the issue.